__tests__/app.test.js

```diff
@@ -140,12 +140,49 @@ describe("Task 5 - GET /api/articles", () => {
 });
 
 describe("Task 6 - GET /api/articles/:article_id/comments", () => {
-  test("should test this function", () => {
+  test("should return an array of objects with the appropriate properties", () => {
     return request(app)
       .get("/api/articles/1/comments")
       .expect(200)
       .then((response) => {
-        expect(response).toBe("broken on purpose - finished for day");
+        const comments = response.body.comments;
+        comments.forEach((comment) => {
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)